### DISCUSSION TITLE
When using browser API, shoud I write one code or one for Chrome and one for Firefox

### DISCUSSION THREAD

**Discussion by oussama-he** [quasarframework/quasar#10750] [CLI - BEX mode]
When developing in the BEX mode, and when using browser API, Does Quasar take care to polyfill the code, or I should write it twice. e.g. Firefox uses browser.tabs and Chrome uses chrome.tabs

**Comment by julie777** [comment]
I am also curious about this. I thought I saw that polyfills were included by default, but in my generated project `chrome` instead of `browser` is in the generated code.
